Readme/automation.md

```diff
@@ -0,0 +1,159 @@
+### 3. Samples
+In this repository, we have already provided samples for executing automation test with Kobiton through TeamCity:
+  -  [automation-test.js](../sample/automation-script/automation-test.js): script written in NodeJS for execute automation test on Kobiton devices.
+  -  [server.rb](../sample/server/server.rb): Script for initiating WebHook server.
+
+To use the provided sample, follow these steps:
+1. Fork this repository https://github.com/kobiton/TeamCity-HockeyApp-Appium-ReactNative.
+2. In your TeamCity project, create a `Build configurations` to use the forked repository.
+3. Clone the forked repository.
+## B. SETUP
+### 1. Getting desired capabilities
+In order to interact with a specific device in Kobiton, its corresponding desired capabilities needs to be supplied.
+
+In the provided sample automation test script, we have pre-configured them to execute automation test of the provided application(s) from `APP_URL` environment variable, and execute the tests on one of available devices which name begins with `iPhone` or `Galaxy` if **ONLY** platform name is supplied in desired capabilities (Android/iOS).
+
+**How to get desired capabilities**
+
+In this tutorial, we will be showing how to get desired capabilities for executing automation test on `Pixel 2 XL 8.1.0`.
+  1. Go to https://portal.kobiton.com and login to your Kobiton account.
+  2. Click **Devices** in the top navigation bar.
+
+  ![click devices](./assets/devices.png)
+ 
+  3. Hover over any device you want to test with and click on Automation settings button (the one with gear symbol).
+ 
+  ![click auto](./assets/click_auto.png)
+ 
+  4. On the left-hand side, you can select your preferred language, App Type, etc.
+  - In this example, we use **NodeJS** as the default language of the script. Therefore, choose **NodeJS** in `Language` section.
+  - Select **Hybrid/Native from Apps** in `App type` section. 
+
+ Kobiton automatically generates your desired capabilities into code based on the selected options. Note values of `deviceName`, `platformversion` and `platformName` to prepare next steps.
+
+  ![desiredcaps](./assets/desiredcaps.png) 
+### 2. Configuring automation test script 
+In the `automation-script` folder, we have provided a sample scripts for executing automation test on Kobiton devices.
+
+As mentioned above, the desired capabilities in provided automation test scripts have already been pre-configured. If you use default configuration, add only a variable named `KOBITON_DEVICE_PLATFORM_NAME` with value `iOS` if you want to execute on iOS or `Android` if on Android.
+
+If you have chosen custom desired capabilities, please follow steps below to apply the collected desired capabilities.
+
+**Setting desired capabilities**
+
+Add there environment variables to TeamCity project:
```

Comment:
Three envs ? The desired capabilities generated on Kobiton has more options, please refer to mine, alongside with the env table